### PR TITLE
#86: refactored SqlTestUtil

### LINKED ISSUE #86: Remove duplicated SqlTestUtil class
### Problem
SqlTestUtil exists in two places: test folder of this project and test folder of virtual-schema-common project. 
We need to remove the duplication lately.

### Acceptance criteria

1. Find out if this is really test code (quick scan indicates it is)
2. If it is test code, move it to test
3. Find out if it needs to be in a static helper class or belongs somewhere else.
4. If it stays in a static helper class:
5. Give that class a proper name (*util* is not acceptable)
6. Hide the constructor
7. Make the class static

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #86

### CODE REVIEW

File: jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void testSqlGenerator() throws AdapterException {
         final SqlGenerationVisitor generator = new TestSqlGenerationVisitor(
                 new ExasolSqlDialect(null, AdapterProperties.emptyProperties()), context);
         final String actualSql = node.accept(generator);
-        assertEquals(SqlTestUtil.normalizeSql(expectedSql), SqlTestUtil.normalizeSql(actualSql));
+        assertEquals(SqlNormalizer.normalizeSql(expectedSql), SqlNormalizer.normalizeSql(actualSql));
```
Comment (redcatbear) [inline jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java]: Use `JSONAssert` instead.

Comment (AnastasiiaSergienko) [reply jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java]: We discussed that it's not a right place to use  JSONAssert

File: jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
 import com.exasol.adapter.sql.*;
 import com.google.common.collect.ImmutableList;
 
-import utils.SqlTestUtil;
+import utils.SqlNormalizer;
```
Comment (redcatbear) [inline jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java]:
```suggestion
import com.exasol.sql.SqlNormalizer;
```

Comment (AnastasiiaSergienko) [reply jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java]: fixed

**Pull request merged**